public/js/index.js

```diff
@@ -1,40 +1,44 @@
 'use strict';
 
+var app = app || {};
 
-(function () {
+
+(function (module) {
 
   function hamburgerClick() {
     $('#nav img').fadeIn();
     $('#hamburger').hide();
   }
 
-  function aboutClick() {
-    $('#listOfProjects').hide();
-    $('#aboutMeContent').fadeIn();
-  }
-
   $('#hamburger').on('click', hamburgerClick);
 
-  $('#aboutMeButton').on('click', aboutClick);
-
-  var rawHtml = $('#handlebars').html();
-  var templateFunction = Handlebars.compile(rawHtml);
-
   function Project (obj) {
     this.name = obj.projName;
     this.location = obj.projLink;
   }
 
+  Project.aboutClick = () => {
+    $('#listOfProjects').hide();
+    $('#aboutMeContent').fadeIn();
+  }
+
+  Project.homeClick = () => {
+    $('#aboutMeContent').hide();
+    $('#listOfProjects').fadeIn();
+  }
+
   Project.prototype.toHtml = () => {
```

Comment:
This is a case where you cannot use an arrow function since you need to bind the context in order for this method to work. Basically, arrow functions cannot be used for prototype methods.